levelkdev/swapr-dapp
# Fix: Display swap button above error message in expert mode

# Summary

Fixes #1068 

Display the swap button above an error message in expert mode

![Selection_173](https://user-images.githubusercontent.com/100695408/179949710-60c48ddb-5f81-45c8-8f82-a21e41c8304c.png)

UPDATE:
I need to pass to the error component prop to identify which variant should we display

![Selection_174](https://user-images.githubusercontent.com/100695408/180004640-6d1a88a9-a512-402d-9a9a-0499addb92ea.png)


  # To Test

1. Open the swap page
2. Start transaction
3. Reject transaction
4. Swap button should be displayed
